pkg/controller/networkpolicy/store/appliedtogroup.go

```diff
@@ -174,6 +174,26 @@ func NewAppliedToGroupStore() storage.Interface {
 			}
 			return []string{atg.Selector.Namespace}, nil
 		},
+		PodIndex: func(obj interface{}) ([]string, error) {
+			atg, ok := obj.(*types.AppliedToGroup)
+			if !ok {
+				return []string{}, nil
+			}
+			keys := make([]string, 0)
```

Comment:
same comment as above

Reply:
It gives a compilation error if I don't specify length